rest-service/manager_rest/rest/resources_v2/plugins.py

```diff
@@ -174,8 +176,11 @@ def get(self, plugin_id, **kwargs):
         """
         # Verify plugin exists.
         plugin = get_storage_manager().get(models.Plugin, plugin_id)
-
-        archive_name = plugin.archive_name
+        # While installing a plugin we add an `installing` prefix to the
+        # archive_name, since the archive is saved without this prefix we
+        # remove it before searching for the archive in the file system.
+        archive_name = (plugin.archive_name[11:] if INSTALLING_PREFIX
```

Comment:
I think it's better to use here the partition function of string for getting the archive name

Reply:
Maybe also move this to common, it can be used also for the removing of the prefix